lib/simulation/openfisca/mapping/index.js

```diff
@@ -100,6 +100,25 @@ var applyRessources = function(src, dest, schema, dateDeValeur) {
     });
 };
 
+var setNonInjectedPrestaToZero = function(dest, dateDeValeur){
+    var moisCourant = moment(dateDeValeur).format('YYYY-MM');
+
+    var prestationsInjectees = _.pick(PRESTATIONS, function(format) {
+
+        return format.type == Number;
+    });
+
+    _.forEach(prestationsInjectees, function(format, nomPresta) {
```

Comment:
Avoid Frenglish + inconsistent domain names: `nomPresta` -> `prestationName` (or something in the same vein).